store/src/main/java/org/apache/rocketmq/store/AllocateMappedFileService.java

```diff
@@ -76,13 +82,14 @@ public MappedFile putRequestAndReturnMappedFile(String nextFilePath, String next
             canSubmitRequests--;
         }
 
+        String nextNextFileName = nextNextFilePath.substring(nextNextFilePath.lastIndexOf("/") + 1);
```

Comment:
It is necessary to use File.separator, as the file path separator on other OS may not be '/'.

Reply:
Okay, fixed now.

Reply:
The UTs also need to be modified accordingly.

Reply:
Sorry, it was my oversight. I have modified it as required. Please review it again. Thank you.

Reply:
> The UTs also need to be modified accordingly.

By the way, is there any compatibility issue here? From the code, it seems that there isn't.

Reply:
> Sorry, it was my oversight. I have modified it as required. Please review it again. Thank you.

By the way, is there any compatibility issue here? From the code, it seems that there isn't.

Reply:
Do you mean file path separator in test case? I also have this doubts but the other use cases I refer to in this file are like this.

Reply:
Sorry, there are indeed some compatibility issues in my UTs,  I have located the problem in the workflows. I will provide a new patch after completing the test in my repository later.

Reply:
> a new patch after completing the test in my repository 
Fixed compatibility issue.

Reply:
It seems like some errors blocked the workflow checks, but I don't have permission to stop or rerun it.